### ISSUE TITLE
Observations from 2021.8.1 automatic release

### ISSUE THREAD

**Issue by jrbourbeau** [dask/dask-docker#180]
A couple unexpected things happened with the latest automatic release procedure:

- [ ] GitHub actions didn't run when the pull request was initially opened (though when I pushed an empty commit, GHA ran as expected). I checked https://www.githubstatus.com/ and there were know reported issues with GitHub actions during this time. xref https://github.com/dask/dask-docker/pull/178
- [ ] Even though the `automerge` label was applied to the PR, it wasn't automatically merged after all CI checks passed. xref https://github.com/dask/dask-docker/pull/178
- [ ] Twice as many `build` CI jobs ran on https://github.com/dask/dask-docker/pull/178. I suspect this is because we're triggering on both `pull_request` and `push` events
https://github.com/dask/dask-docker/blob/e82ac06b362319833954ae64687061606f22b924/.github/workflows/build.yml#L3
- [ ] The PR which was automatically created updated the `release: XXXX.XX.X` field in `build/docker-compose.yml` but not `.github/workflows/build.yml` (xref https://github.com/dask/dask-docker/pull/179)
- [ ] Images for the new 2021.8.1 release aren't appearing on docker hub (https://hub.docker.com/r/daskdev/dask/tags?page=1&ordering=last_updated)

**Comment by jrbourbeau** [comment]
For reference, since our `build` workflow will push to docker hub on `push` events with a new tag

https://github.com/dask/dask-docker/blob/3362618f21512dfb66a42877ccd846375e611a66/.github/workflows/build.yml#L78

I ended up manually deleting the 2021.8.1 tag (which was generated from the `autotag` workflow) and pushing an empty commit to `main` with a new 2021.8.1 tag. This resulted in docker images for the new release being published

**Comment by jacobtomlinson** [comment]
Thanks for writing this up @jrbourbeau. I'll take a look at these.

**Comment by jakirkham** [comment]
Curious how many of these are still issues

**Comment by jrbourbeau** [comment]
IIUC all of the unchecked boxes in the original post are still valid

**Comment by jacobtomlinson** [comment]
I've fixed up a few things and checked a few boxes in the post above that I can confirm as resolved.

I'm keen to wait until the next release to see if they are resolved or not. cc @jrbourbeau @jsignell.

**Comment by jacobtomlinson** [comment]
ping @jrbourbeau @jsignell how did things go with the latest release?

I see some extra changes were pushed to the PR so I'm guessing the find and replace still isn't working quite right, and the automerge didn't work. Is there anything else?

**Comment by jrbourbeau** [comment]
Yeah, that's correct. I also noticed that the autotag workflow did assign a `2022.4.0` tag, but assigning that tag didn't result in us pushing Docker images to Docker Hub or ghcr.io. As of now the latest images available are for `2022.3.0`. I should also note that deleted the `2022.4.0` tag which was automatically added and manually re-added the tag myself to see if that would help, but it did not
